.github/ROADMAP.md

```diff
@@ -12,11 +12,10 @@ The team will release one minor version upgrade each quarter, or whenever a new
 
```

Comment:
Can we add object-tracking algorithm, ie. deep-sort, optical-flow? (at least for 2d object detection task in this phase.)

Reply:
Some suggested VOS task are related